pkg/fftypes/config.go

```diff
@@ -0,0 +1,22 @@
+// Copyright © 2021 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fftypes
+
+type ConfigRecord struct {
```

Comment:
As there is a `512` limit on the database record, it would be nice to have a `Validate` method on this that does a check length. Not a blocker to merge, as (unlike some other cases) it doesn't matter if we defer down to the DB for an error. But it would be more friendly to have an error at the API layer